lib/consolelogger.js

```diff
@@ -23,6 +23,10 @@ function getTS() {
     " " + hour + ":" + min + ":" + sec;
 }
 
+ConsoleLogger.prototype.setLogLevel = function(level) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)